mne_bids/datasets.py

```diff
@@ -38,3 +41,68 @@ def fetch_faces_data(data_path, repo, subject_ids):
             tf.extractall(path=data_path)
             shutil.move(op.join(data_path, 'ds117'), target_dir)
             os.remove(tar_fname)
+
+
+def download_matchingpennies_subj(url_dict=None, directory='.'):
+    """Download and save subject data from the matching pennies dataset[1].
+
+    Parameters
+    ----------
+    url_dict : dict | None
+        Dictionary with keys corresponding to the type of BIDS file
+        and its extension. Dict values corresponding to valid urls
+        pointing to the respective files on the Open Science
+        Framework. MUST contain a key value pair
+        "subj_id: int" (see Examples). If None, will download data for sub-05.
+    subj_id : int | None
+        The subject identifier the url_dict corresponds to. If None,
+        will be set to 99.
```

Comment:
if it downloads `sub-05` by default, why not set this to `05`?

Reply:
it could be that a user specifies
```python
{'channels.tsv': 'https://osf.io/wzyh2/',
 'eeg.eeg': 'https://osf.io/mj7q8/'}
```
Where the links point to some other subject or not subject at all ... but I agree, this is a bit weird ... I'll think about a fix.